Formula/cairo.rb

```diff
@@ -47,6 +54,10 @@ def install
       args << "--enable-xcb=no" << "--enable-xlib=no" << "--enable-xlib-xrender=no"
     end
 
+    if build.head?
+      system "./autogen.sh", *args
+    end
+
```

Comment:
Also, the arguments that are being passed to `./configure` (the ones in `args`) are probably not the ones you will want to pass to `./autogen.sh`?!